src/components/BookReviews/BookReviews.jsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react"
+import './BookReviews.css'
+
+function BookReviews({id, reviews, setReviews}){
```

Comment:
I'm going to be pedantic and ask you to rename this component - I'm happy with all the others, but this one is inconsistent with your other form components, and doesn't actually describe what it is.

BookReviewForm or something would be miles better.

Don't orget to also rename the file and folder along with any imports